assets/code-editor.js

```diff
@@ -106,12 +106,13 @@ angular.module('code.editor', [])
 	'				<label class="control-label">Change CFML Engine</label>'+
 	'		        <div>'+
 	'		         <select id="engine" class="form-control">'+
-	'		             <option value="lucee5">Lucee 5.LATEST</option>'+
-	'		             <option value="lucee">Lucee 4.5.LATEST</option>'+
-	'		             <option value="railo">Railo 4.2</option>'+
-	'				     <option value="acf2016">Adobe ColdFusion 2016</option>'+
-	'				     <option value="acf11">Adobe ColdFusion 11</option>'+
-	'		             <option value="acf">Adobe ColdFusion 10</option>'+
+	'		             	<option value="lucee5">Lucee 5.LATEST</option>'+
+	'		             	<option value="lucee">Lucee 4.5.LATEST</option>'+
+	'		             	<option value="railo">Railo 4.2</option>'+
+	'				<option value="acf2018">Adobe ColdFusion 2018</option>'
+	'				<option value="acf2016">Adobe ColdFusion 2016</option>'+
+	'				<option value="acf11">Adobe ColdFusion 11</option>'+
+	'		             	<option value="acf">Adobe ColdFusion 10</option>'+
```

Comment:
You also need to add a URL for cf2018 look around line 213

Reply:
Looks like that URL should be https://acf13-sbx.trycf.com/getremote.cfm

Reply:
This url is not available. Neither with acf13 nor acf18 in subdomain.